jetbrains-core/src/software/aws/toolkits/jetbrains/services/codewhisperer/explorer/CodeWhispererExplorerActionManager.kt

```diff
@@ -267,6 +298,12 @@ internal class CodeWhispererExploreActionState : BaseState() {
     // can not remove this as we want to support existing accountless users
     @get:Property
     var token by string()
+
+    @get:Property
+    var timestamp by string()
+
+    @get:Property
+    var doNotShowAgain by string()
```

Comment:
since it's a Boolean type, let's put it into CodeWhispererExploreStateType and `value` map (Line 296)

Reply:
done, thanks!